source/isaaclab/isaaclab/actuators/actuator_cfg.py

```diff
@@ -36,42 +36,73 @@ class ActuatorBaseCfg:
     This limit is used to clip the computed torque sent to the simulation. If None, the limit is set to the value
     specified in the USD joint prim.
 
+    .. attention::
+
+        The :attr:`effort_limit_sim` attribute should be used to set the effort limit for the simulation physics
+        solver.
+
+        The :attr:`effort_limit` attribute is used for clipping the effort output of the actuator model *only*
+        in the case of explicit actuators, such as the :class:`~isaaclab.actuators.IdealPDActuator`.
+
     .. note::
 
-        For ImplicitActuators this value will be collapsed with effort_limit_sim due to duplicating functionality. If
-        both are set the effort_limit_sim will be used priority.
+        For implicit actuators, the attributes :attr:`effort_limit` and :attr:`effort_limit_sim` are equivalent.
+        However, we suggest using the :attr:`effort_limit_sim` attribute for its clarity.
+
     """
 
     velocity_limit: dict[str, float] | float | None = None
     """Velocity limit of the joints in the group. Defaults to None.
 
     This limit is used by the actuator model. If None, the limit is set to the value specified in the USD joint prim.
 
-    .. note::
+    .. attention::
+
+        The :attr:`velocity_limit_sim` attribute should be used to set the velocity limit for the simulation physics
+        solver.
 
-        velocity_limit is not used in ActuatorBaseCfg but is provided for inherited version like
-        :class:`isaaclab.actuators.DCMotor`.
+        The :attr:`velocity_limit` attribute is used for clipping the effort output of the actuator model *only*
+        in the case of explicit actuators, such as the :class:`~isaaclab.actuators.IdealPDActuator`.
```

Comment:
```suggestion
        in the case of explicit actuators, such as the :class:`~isaaclab.actuators.DCMotor`.
```
IdealPDActuator does not use this attribute for clipping effort, only the DCMotor

Reply:
IdealPD uses effort limit when it calls _clip_effort...

Reply:
Right but this is commenting on velocity_limit not the effort_limit.